debian/migrate-collect-to-home.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
I think we can remove this script. The collect app has been removed 5 or 6 years ago.

Reply:
Removed